languages/elixir/exercises/concept/recursion/.meta/example.ex

```diff
@@ -0,0 +1,18 @@
+defmodule BirdCount do
+  def today([]), do: nil
+  def today([head | _]), do: head
+
+  def increment_day_count([]), do: [1]
+  def increment_day_count([head | tail]), do: [head + 1 | tail]
+
+  def has_day_without_birds?([]), do: false
+  def has_day_without_birds?([0 | _]), do: true
+  def has_day_without_birds?([_ | tail]), do: has_day_without_birds?(tail)
+
+  def total([]), do: 0
+  def total([head | tail]), do: head + total(tail)
+
+  def busy_days([]), do: 0
+  def busy_days([head | tail]) when head >= 5, do: busy_days(tail) + 1
+  def busy_days([_ | tail]), do: busy_days(tail)
```

Comment:
None of those functions force you to use an accumulator. I was wondering if it would be a good idea to add some function that will force you to use an accumulator, or is that too advanced?

Reply:
At one point, there was some discussion about a tail-call optimization exercise proposed by @fireproofsocks. If that were created, it could be a good place for the discussion about accumulators. If it isn't, it could be good to have this here as well.

Reply:
Since there is an issue for a `tail-recursion` concept exercise, I am assuming it will be created at some point and it will also cover accumulators.